src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -39,11 +39,12 @@
     },
     {
         "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask",
-        "version": "2.1.1",
+        "version": "2.2.0",
         "name": "DurableTask",
         "bindings": [
             "activitytrigger",
             "orchestrationtrigger",
+            "entityTrigger",

```

Comment:
Can you make this lower case, i don't think i should make a difference. But i need to check the logic.